CHANGELOG.md

```diff
@@ -79,6 +79,7 @@
     Feature #3276: Editor: Search- Show number of (remaining) search results and indicate a search without any results
     Feature #3641: Editor: Limit FPS in 3d preview window
     Feature #3703: Ranged sneak attack criticals
+    Feature #4012: OpenMW-CS: Create log file like with OpenMW
```

Comment:
Changelog already has this line, though in wrong oder. I'd prefer that you don't touch the changelog because then I'd have to solve merge conflicts in the neighboring PR again (I placed the line where it belongs there).